packages/flutter_tools/lib/src/web/web_device.dart

```diff
@@ -244,7 +341,7 @@ class WebServerDevice extends Device {
   Future<bool> get isLocalEmulator async => false;
 
   @override
-  bool isSupported() => featureFlags.isWebEnabled;
+  bool isSupported() => true;
```

Comment:
If featureFlags.isWebEnabled == false, then the device discoverer isn't supported which is redundant with this check

Reply:
I understand that currently this check is redundant. Could a future refactor change that however?

Reply:
There is an existing workflow test that confirms that the web devices aren't included if the config is disabled: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/general.shard/web/workflow_test.dart#L57

I added another test case to the `web/devices_test` that verifies no devices are returned if the config is disabled